contrib/download-frozen-image-v2.sh

```diff
@@ -56,6 +56,13 @@ while [ $# -gt 0 ]; do
 	layersFs=$(echo "$manifestJson" | jq --raw-output '.fsLayers | .[] | .blobSum')
```

Comment:
Do you think it'd be cleaner to just add a `| tr -d '\r'` here?  It should be harmless on other platforms to just remove `\r`s entirely, and would let us keep the block below unencumbered with version testing.

Sorry for the super-late, post-merge review. :disappointed: